ProjFS.Mac/PrjFSKext/Message_Kernel.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "public/Message.h"
+
+void Message_Init(
+    Message* spec,
+    MessageHeader* header,
+    uint64_t messageId,
+    MessageType messageType,
+    const FsidInode& fsidInode,
+    int32_t pid,
+    const char* procname,
+    const char* path,
+    const char* fromPath);
+
+uint32_t Message_Encode(void* buffer, uint32_t bufferSize, const Message& message);
```

Comment:
I've now moved the kernel-only function prototypes into a kernel-only header file.